### ISSUE TITLE
the bug with  <android:fontFamily="sans-serif-light">

### ISSUE THREAD

**Issue by FlogFly** [techery/progresshint#19]
I run your sample ,but when app:popupLayout use  the default layout <proress_hint_popup.xml>  ,  it has a bug , the popup ' s num  not change  . but if <Textview> has android:fontFamily="sans-serif-light" , the num will change .  it's why the <seekbar_vertical2（yellow background）> in the sample can change num ,because the layout 's textview has android:fontFamily="sans-serif-light" 

![](https://upload-images.jianshu.io/upload_images/1529568-2cd823a68fc68243.gif?imageMogr2/auto-orient/strip)

**Comment by FlogFly** [comment]
![ezgif com-optimize](https://user-images.githubusercontent.com/8286007/44779112-2dc13d00-abb1-11e8-958c-f1d902217cf5.gif)

**Comment by almozavr** [comment]
@FlogFly I can't reproduce the issue, let's try find out what differs:

* what's the version of android system you found out the issue at?
* does it occur on android emulator or specific device only?

**Comment by almozavr** [comment]
I've also updated dependencies to the latest -- so far so good
```
buildscript {
  repositories {
    google()
    jcenter()
  }
  dependencies {
    classpath 'com.android.tools.build:gradle:3.1.4'
    classpath 'com.github.dcendents:android-maven-gradle-plugin:2.0'
  }
}

ext {
  compileSdkVersion = 27
  buildToolsVersion = '28.0.2'
  minSdkVersion = 15
  targetSdkVersion = 27

  supportPackageVersion = '27.1.1'
}

allprojects {
  repositories {
    jcenter()
    google()
  }
}
```

**Comment by almozavr** [comment]
I'm closing the issue -> it's not reproducable and possibly fixed with androidX, see release `0.4.+`
